inc/Plugin.php

```diff
@@ -96,13 +96,37 @@ public function register_hooks(): void {
 						$locale  = GP_Locales::by_slug( $translation_set->locale );
 						$project = GP::$project->get( $translation_set->project_id );
 
-						return sprintf(
+						/**
+						 * Filters whether a Slack notification for translation updates from GitHub should be sent.
+						 *
+						 * @since 2.0.3
+						 *
+						 * @param bool               $send_message    Whether to send a notification or not. Default true.
+						 * @param GP_Translation_Set $translation_set Translation set the ZIP is for.
+						 * @param GP_Project         $project         The GlotPress project that was updated.
+						 */
+						$send_message = apply_filters( 'traduttore_zip_generated_send_notification', true, $translation_set, $project );
+
+						if ( ! $send_message ) {
+							return false;
+						}
+
+						$message = sprintf(
 							'<%1$s|%2$s>: ZIP file updated for *%3$s*. (<%4$s|Download>)',
 							home_url( gp_url_project( $project ) ),
 							$project->name,
 							$locale->english_name,
 							$zip_url
 						);
+
+						/**
+						 * Filters the Slack notification message when a new translation ZIP file is built.
+						 *
+						 * @since 2.0.3
+						 *
+						 * @param string $message The notification message.
+						 */
+						return apply_filters( 'traduttore_zip_generated_notification_message', $message );
```

Comment:
Should probably pass the same arguments as for the default $message or ` $zip_path, $zip_url, $translation_set`.